scripts/dts/gen_defines.py

```diff
@@ -440,7 +440,14 @@ def irq_name_alias(irq, cell_name):
         # We ignore the controller for now
         for cell_name, cell_value in irq.specifier.items():
             ident = "IRQ_{}".format(irq_i)
-            if cell_name != "irq":
+            if cell_name == "irq":
+                # TODO: ADD COMMENT ABOUT ZEPHYR IRQ NUM
+                irq_ctrl = irq.controller
+                while irq_ctrl.interrupts:
+                    cell_value <<= 8
+                    cell_value |= irq_ctrl.interrupts[0].specifier['irq']
```

Comment:
Nit: Double quotes for consistency.

Should add error checking and not assume that the "irq" key exists. Maybe some kind of helper function for the `cell_name == "irq"` case would work.

Reply:
Fixed the quote's.  Added a helper function.

What were you thinking w/regards to the error checking?

Reply:
Just that "irq" isn't known to be in there (if someone messed up the bindings). Should fail with something nicer than a `KeyError: 'irq'` from Python in that case.